packages/canary-web-components/src/components/ic-data-table/ic-data-table.tsx

```diff
@@ -2158,8 +2155,7 @@ export class DataTable {
           class={{
             "loading-empty": isLoading,
             loading: true,
-            "show-background":
-              loadingOptions.showBackground || loadingOptions.overlay,
+            "show-background": true,
```

Comment:
Could the styling in this class just be absorbed by the .loading class? Since its now the default behaviour?